lua/caching_handlers.lua

```diff
@@ -15,22 +15,50 @@ function caching_handlers._extract_ids_from_uri(uri, pattern)
     return ids
 end
 
--- Callback to save response to cache, to be executed after the response has been sent
-function caching_handlers._post_request_callback(response, request_info, cacheability_info)
+-- Function to compute the id field used in the cache.
+function caching_handlers._get_cache_key(request_info, cacheability_info)
```

Comment:
Since it returns only the ids, shouldn't this be `_get_ids`? This name makes it sound like we're returning the entire cache key

Reply:
renamed to _get_cache_id